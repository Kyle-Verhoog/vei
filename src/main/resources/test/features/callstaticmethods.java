public class Callstaticmethods {
  public Callstaticmethods() {}
  public void m2() {
    java.lang.System.exit();
  }
}