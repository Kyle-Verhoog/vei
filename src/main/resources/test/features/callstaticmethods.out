PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
) ┃ )
{ ┃ {
} ┃ }
PUBLIC ┃ public
VOID ┃ void
IDENTIFIER ┃ m2
( ┃ (
) ┃ )
{ ┃ {
IDENTIFIER ┃ java
. ┃ .
IDENTIFIER ┃ lang
. ┃ .
IDENTIFIER ┃ System
. ┃ .
IDENTIFIER ┃ exit
( ┃ (
) ┃ )
; ┃ ;
} ┃ }
} ┃ }
